app/common/lib/shieldUtil.js

```diff
@@ -0,0 +1,20 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const frameStateUtil = require('../../../js/state/frameStateUtil')
```

Comment:
shieldState is fine, but I'd still like to pass in the frame for now vs `activeRequestedLocation`. Can we also change the name to `braveShieldsEnabled`? !braveShieldsDisabled is harder to read